src/Cookie.php

```diff
@@ -66,11 +66,13 @@ class Cookie {
 	/**
 	 * Create a new cookie object
 	 *
-	 * @param string $name
-	 * @param string $value
+	 * @param string                                                  $name           The name of the cookie.
+	 * @param string                                                  $value          The value for the cookie.
 	 * @param array|\WpOrg\Requests\Utility\CaseInsensitiveDictionary $attributes Associative array of attribute data
-	 * @param array $flags
-	 * @param int|null $reference_time
+	 * @param array                                                   $flags          The flags for the cookie.
+	 *                                                                                Valid keys are (currently) creation,
+	 *                                                                                last-access, persistent and host-only.
```

Comment:
Not sure about the `(currently)` here. Isn't all of the source code - including the docblocks - just the current version of the library?

Reply:
True. To be fair, I copied this text (including the `(currently)` from the docblock for the property to which the `$flags` parameter will be assigned, so I think I should probably removed it from both in that case.